library/src/main/java/com/xfinity/blueprint/ViewAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.xfinity.blueprint
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.SCROLL_STATE_IDLE
+import com.xfinity.blueprint.model.Component
+
+class ViewAdapter(private val recyclerView: RecyclerView,
+                  private var loadingView: View? = null) : ViewRegistry {
+
+    override fun updateComponents(newComponents: List<Component>, behavior: () -> Unit) {
```

Comment:
You don't seem to update anything. newComponents param is in no use too.

Reply:
Updates are happening. I agree that newComponents param looks redundant in this case.